src/lcmap/chipmunk/setup.clj

```diff
@@ -69,25 +69,6 @@
                                                :extra    :text})))
 
 
-(defn create-inventory-tile-index
-  ""
-  []
-  (hayt/create-index :inventory
-                     :tile
-                     (hayt/index-name :inventory_tile_ix)
-                     (hayt/if-exists false)))
-
-
-(defn create-inventory-layer-index
-  ""
-  []
-  (hayt/create-index :inventory
-                     :layer
-                     (hayt/index-name :inventory_layer_ix)
-                     (hayt/if-exists false)))
-
-
-
 (defn create-inventory-materialized-view
```

Comment:
i wonder, if chipmunk isn't actually enabling queries of this MV, should it be responsible for creating it?